fw-child/resources/scss/child.scss

```diff
@@ -963,3 +954,20 @@ $query-card-hover-padding: 0.75rem;
 //
 
 @import 'pages';
+
+//
+// Pagination
+//
+
+@media (max-width: 1000px) {
+  .fw-query-pagination {
+    flex-wrap: wrap;
+
+    &-pages {
```

Comment:
@xfrenette-crim I found that this will create some inconsistency on the UI in regard the prev/next buttons.

<img width="413" alt="News_–_Climate_Data_Canada" src="https://github.com/CanadianClimateDataPortal/climatedata-wp-theme/assets/116840223/c046ca75-874d-4323-8931-b0da16b69f9e">